adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/CoapAdapterProperties.java

```diff
@@ -23,16 +23,52 @@
 public class CoapAdapterProperties extends ProtocolAdapterProperties {
 
     /**
-     * The default regular expression to split the identity into authority and tenant.
+     * The default regular expression to split the identity into authority and tenant. Value
```

Comment:
There is no need for the `@value` tag here as this is already the place where it is defined.
Using the `@value` tag makes sense in JavaDoc comments of other methods where you would like to refer to the value of the constant instead of the definition (as you already do below) ...

Reply:
OK